app/src/main/java/com/example/marcinwisniewski/intivetrainingapp/movielist/viewmodel/ListViewModel.kt

```diff
@@ -0,0 +1,22 @@
+package com.example.marcinwisniewski.intivetrainingapp.movielist.viewmodel
+
+import android.arch.lifecycle.MutableLiveData
+import android.arch.lifecycle.ViewModel
+import com.example.marcinwisniewski.intivetrainingapp.movielist.model.Movie
+import com.example.marcinwisniewski.intivetrainingapp.movielist.model.MovieProvider
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.schedulers.Schedulers
+
+class ListViewModel : ViewModel() {
+    fun getMovies(): MutableLiveData<List<Movie>> {
+        val moviesList = MutableLiveData<List<Movie>>()
+        MovieProvider.movieProvider.getMoviesResponse()
+                .subscribeOn(Schedulers.io())
+                .observeOn(AndroidSchedulers.mainThread())
+                .subscribe({
+                    moviesList.value = it.data
+                }) {
+                }
```

Comment:
Add TODO to handle error

Reply:
Will be add on the first commit on new feature branch